recipe/meta.yaml

```diff
@@ -5,35 +5,32 @@ package:
   version: {{ version }}
 
 source:
-  url: https://support.hdfgroup.org/ftp/HDF/releases/HDF{{ version }}/src/hdf-{{ version }}.tar.bz2
-  sha256: bde035ef5a1cd5fdbd0a7f1fa5c17e98bbd599300189ac4d234f16e9bb7bcb12
-  patches:
-    - patches/max_files.patch
-    - patches/0001-Don-t-build-ctest0-and-run-ncgen-when-not-testing.patch
-    - patches/debian/64bit.patch
-    - patches/debian/hdf-4.2.10-aarch64.patch
-    - patches/debian/hdfi.h.patch
-    - patches/debian/spelling-errors.patch
+  - git_url: https://github.com/HDFGroup/hdf4
+    git_rev: hdf-{{ version|replace(".", "_") }}
+    git_depth: 1
+    patches:
+      - patches/max_files.patch
+      - patches/0001-Don-t-build-ctest0-and-run-ncgen-when-not-testing.patch
+      - patches/debian/64bit.patch
+      - patches/debian/hdf-4.2.10-aarch64.patch
+      - patches/debian/hdfi.h.patch
+      - patches/debian/spelling-errors.patch
 
 build:
-  number: 3
+  number: 4
   run_exports:
     # change SONAMEs with bugfix rev.
     #    https://abi-laboratory.pro/tracker/timeline/hdf5/
     - {{ pin_subpackage('hdf4', max_pin='x.x.x') }}
 
 requirements:
   build:
-    - cmake       # [win]
+    - cmake
     - bison       # [not win]
     - flex        # [not win]
-    - make        # [not win]
```

Comment:
`make` is still necessary I think

Reply:
I was assuming cmake brought that with it but you are correct that it doesn't.